README.md

```diff
@@ -97,7 +109,7 @@ Below is a comprehensive list showing all zbox commands along with their respect
+| updateallocation | Update Allocation Settings.<br /><br />[Update allocation with free storage marker](#update-allocation): Update allocation settings with a free storage marker.<br /><br />[Update allocation size](#update-allocation): Update allocation size default is 2GB.<br /> <br />[Forbid operations on allocation](#update-allocation): Update allocation settings and forbid operations such as copy,update,delete,move,rename and upload.<br /><br />[Add Blobber](#add-blobber): Add a blobber to an allocation for cdn purposes.<br /><br />[Replace Blobber](#replace-blobber): Add or remove a blobber from existing allocation to prevent vendor lock-in. | Update allocation with free storage marker:`./zbox updateallocation --allocation d0939e912851959637257573b08c748474f0dd0ebbc8e191e4f6ad69e4fdc7ac --free_storage "markers/my_marker.json"`<br /><br />Update allocation size:`./zbox updateallocation --allocation d0939e912851959637257573b08c748474f0dd0ebbc8e191e4f6ad69e4fdc7ac --size 4096`<br /><br />Forbid operations on allocation:`./zbox updateallocation --allocation $ALLOC --forbid_upload`<br /><br />Unforbid operations on allocation:`./zbox updateallocation --allocation $ALLOC --forbid_upload false`<br /><br />Add Blobber:`./zbox updateallocation --allocation $ALLOC --add_blobber 98f14362f075caf467653044cf046eb9e8a5dfee88dc8b78cad1891748245003`<br /><br />Replace Blobber:`./zbox updateallocation --allocation $ALLOC --add_blobber 8d19a8fd7147279d1dfdadd7e3ceecaf91c63ad940dae78731e7a64b104441a6 --remove_blobber 06166f3dfd72a90cd0b51f4bd7520d4434552fc72880039b1ee1e8fe4b3cd7ea` |
```

Comment:
Move Forbid operations on allocation Description a little bit up so it can be properly aligned with its usage text